tests/test_definitions/column_aggregate_expectations/expect_column_stdev_to_be_between.json

```diff
@@ -83,7 +83,7 @@
             "observed_value": 1.1547005383792517
           },
           "only_for": [
-            "bigquery_cfe"
+            "bigquery_v3_api"
```

Comment:
Any reason this wasn't just `bigquery` to begin with before?

Reply:
Good question, reading through `test_expectations_v2_api.py` and `test_expectations_v3_api.py` it is used to skip these tests for the v2 api. I think we will have to leave these in until v2 is removed.

Reply:
Ah, this was a relatively new addition back in August in https://github.com/great-expectations/great_expectations/pull/5704

In the self_check/util.py file a lot of the logic for checking only_for and suppress_test_for files in test_definitions was taken from those test_expectations*.py files and I definitely didn't remember any special bigquery cases (only pandas)

See https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/self_check/util.py#L2519-L2594